tasks/gulp-apidoc.js

```diff
@@ -0,0 +1,22 @@
+import gulp from 'gulp';
+import clean from 'rimraf';
+import apidoc from 'apidoc';
+
+const APIDOC_DEST_PATH = './website/public/apidoc';
+const APIDOC_SRC_PATH = './website/src';
+gulp.task('apidoc:clean', (done) => {
+  clean(APIDOC_DEST_PATH, done);
+});
+
+gulp.task('apidoc', ['apidoc:clean'], (done) => {
+  let result = apidoc.createDoc({
+    src: APIDOC_SRC_PATH,
+    dest: APIDOC_DEST_PATH,
+  });
+
+  if (result === false) {
```

Comment:
More or less I copied this from grunt-apidoc, but since it's official I think we can trust them